modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/nested.yml

```diff
@@ -194,3 +194,26 @@ setup:
   - match: { aggregations.courses.highpass_filter.unnest.department.buckets.0.doc_count: 1 }
   - match: { aggregations.courses.highpass_filter.unnest.department.buckets.1.key: math }
   - match: { aggregations.courses.highpass_filter.unnest.department.buckets.1.doc_count: 1 }
+---
+"Illegal reverse nested aggregation to a child nested object":
+  - do:
```

Comment:
This'll also want a capability. Those live in `SearchCapabilities`. And, of course, it's not an `enum`.